### ISSUE TITLE
Can't control the color

### ISSUE THREAD

**Issue by PetabitsProd** [sahilchaddha/homebridge-yeelight-platform#18]
Hi,
The plug-in is nearly perfect but the little problem is that I can't control the color when I use my color preferences it put the color of the light on a yellow or white color.

Is there a configuration that I misunderstood ?

![IMG_15DA42BA4675-1](https://user-images.githubusercontent.com/23233335/56068495-57fdee80-5d7f-11e9-94ed-af5a9218e318.jpeg)

**Comment by anakinsk** [comment]
Not sure if that is what topic starter is talking about, but after recent iOS and OS X updates, I can't set the colour via Home app, it sets the right colour for a second, and then immediately changes it to one of the "color temperature" colours, like white or yellow. Eve app still works okay tho.

**Comment by iChUdY** [comment]
@anakinsk yes this problem really exist, but this plugin probably not developed further.. so bad luck 🤭

**Comment by PetabitsProd** [comment]
@anakinsk Yes this is exactly what i'm talking about, i try to change the color but it changes it immediately to a color temperature, did you find another plug-in to use yeelight bulb (with homebridge) ?

**Comment by manosioa** [comment]
@sahilchaddha Hi, any news about this?

**Comment by manosioa** [comment]
hey @sahilchaddha 
here's my config 

[config.json.zip](https://github.com/sahilchaddha/homebridge-yeelight-platform/files/3258849/config.json.zip)

My Yeelight bulbs are yeelight v2 e27 RGB

If you need anything else please let me know

**Comment by PetabitsProd** [comment]
I can now say that this issue will be close with iOS 13 I am testing the beta and it work very well now

**Comment by manosioa** [comment]
I’m on iOS 13 beta 2 but the problem still exists

**Comment by PetabitsProd** [comment]
Oh I'm sorry I forgot that I change the plugin and not using it anymore... I now use the [yeelight-wifi ](https://github.com/vieira/homebridge-yeelight-wifi) plugin and it works for me maybe we should check how they are doing to change the color properly
